frontend/src/App.vue

```diff
@@ -1,67 +1,76 @@
 <script setup>
-import { ref } from "vue";
 import PersonalInfo from "./components/PersonalInfo.vue";
-import Education from "./components/Education.vue";
 import Summary from "./components/Summary.vue";
-import Experience from "./components/Experience.vue";
-import Skills from "./components/Skills.vue";
-import "./styles/style.css";
-
-import jsPDF from "jspdf";
-import html2canvas from "html2canvas";
-
-const resumeContent = ref(null);
-
-const submitForm = () => {
-  console.log("Form data submitted!");
-};
-
-const exportToPDF = async () => {
-  try {
-    const pdf = new jsPDF("p", "mm", "a3");
-    const pdfWidth = pdf.internal.pageSize.getWidth();
-    const pdfHeight = pdf.internal.pageSize.getHeight();
-
-    const canvas = await html2canvas(resumeContent.value, {
-      scale: 2,
-      dpi: 1080,
-    });
-    const imgData = canvas.toDataURL("image/png");
-    const imgWidth = (canvas.width * pdfWidth) / canvas.width;
-    const imgHeight = (canvas.height * pdfHeight) / canvas.height;
-
-    pdf.addImage(imgData, "JPEG", 0, 0, imgWidth, imgHeight);
-    pdf.save("resume.pdf");
-  } catch (error) {
-    console.error("Error exporting to PDF:", error);
-  }
-};
+import EducationsList from "./components/EducationsList.vue";
+import ExperiencesList from "./components/ExperiencesList.vue";
+import SkillsList from "./components/SkillsList.vue";
+import Menu from "./components/Menu.vue";
+import "./style.css";
 </script>
 
 <template>
```

Comment:
this is just me picking out, but it's better to have same code structure in files.
if you decided to have this structure: (as I see the majority of files have it) 

```
<template/>
<script/>
<style/>
```

then it's better to continue to maintain the project following the pattern you have